### PR TITLE
Merged upload-user-files with master branch on local

### PR DESCRIPTION
**Pull request by Keerthi4308**
Hello Marcus,

As suggested resolved merge conflicts in views.py and removed file generated on migration in wagtailapps.

Thanks,
Keerthi

### CODE REVIEW

**Comment by DImuthuUpe** [comment]
@Keerthi4308 as mentioned above, your changes have conflicts with current master. Do a rebase and update the PR

File: django_airavata.log [hunk 1]
```diff
@@ -0,0 +1,384 @@
+[2019-04-16 14:58:52,674 django_airavata.apps.api.urls:80 DEBUG] router url: <RegexURLPattern project-list ^projects/$>
```
Comment (machristie) [inline django_airavata.log]: Please remove this file from the PR.

File: django_airavata/apps/api/views.py [hunk 2]
```diff
@@ -734,6 +737,147 @@ def get(self, request, format=None):
             data_product, context={'request': request})
         return Response(serializer.data)
 
+experiment_data_storage = FileSystemStorage(
```
Comment (machristie) [inline django_airavata/apps/api/views.py]: experiment_data_storage isn't needed? Also get_user_files and the other functions are duplicated here.

File: django_airavata/apps/workspace/static/django_airavata_workspace/js/components/experiment/ExperimentEditor.vue [hunk 3]
```diff
@@ -166,7 +167,8 @@ export default {
     return {
       projects: [],
       localExperiment: this.experiment.clone(),
-      invalidInputs: []
+      invalidInputs: [],
+      input_name:''
```
Comment (machristie) [inline django_airavata/apps/workspace/static/django_airavata_workspace/js/components/experiment/ExperimentEditor.vue]: This isn't used, and in fact, all of the changes to this file are unnecessary as far as I can tell.

File: django_airavata/wagtailapps/base/migrations/0020_auto_20190416_1644.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
```
Comment (machristie) [inline django_airavata/wagtailapps/base/migrations/0020_auto_20190416_1644.py]: Please remove this file.

Comment (Keerthi4308) [reply django_airavata/wagtailapps/base/migrations/0020_auto_20190416_1644.py]: Hello Marcus,

I applied the requested changes above and from Travis CI. Please let me know if I need to close my previous PR.

Thanks,
Keerthi

**Comment by machristie** [comment]
Hi @Keerthi4308 ,

This is looking really good. Yes, I think you can close the other PR.

I see I've introduced some new changes that have resulted in new merge conflicts 😄 . Don't worry about that. I can fix that up on the merge.

One question: are you still looking into the jupyter-integration API and incorporating that?

**Comment by machristie** [comment]
Thanks for your contribution @Keerthi4308 !

**Comment by Keerthi4308** [comment]
Thank you so much Marcus for your support.

**Pull request merged**